### PR TITLE
Fix routes

### PR DESCRIPTION
**Pull request by Cempl**
Reved old routes

### CODE REVIEW

File: sdk/blockchain.go [hunk 1]
```diff
@@ -37,28 +36,6 @@ func (b *BlockchainService) GetBlockByHeight(ctx context.Context, height Height)
 	return dto.toStruct()
 }
 
-// returns Transaction's inside of block at passed height
-func (b *BlockchainService) GetBlockTransactions(ctx context.Context, height Height) ([]Transaction, error) {
```
Comment (xgreenx) [inline sdk/blockchain.go:41]: Why did you remove this route?) We didn't remove it.

Comment (Wondertan) [reply sdk/blockchain.go:41]: @Cempl

Comment (Cempl) [reply sdk/blockchain.go]: Sorry, my mistake. Rollback is done)

**Review by Wondertan — APPROVED** [review]

**Review by xgreenx — APPROVED** [review]
